action.yml

```diff
@@ -117,7 +117,7 @@ runs:
 
         if [ -n "${{ inputs.extra-conf }}" ]; then
           if [ -n "${{ inputs.github-token }}" ]; then
-            export NIX_INSTALLER_EXTRA_CONF="${{ inputs.extra-conf }}access-tokens = github.com=${{ inputs.github-token }}"
+            export NIX_INSTALLER_EXTRA_CONF="${{ inputs.extra-conf }}"$'\n'"access-tokens = github.com=${{ inputs.github-token }}"
```

Comment:
Hmm.. I wonder if this will work with the `https://` prefix we'd get from: the  `github.server_url` input: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
I think it's fine: https://github.com/NixOS/nix/blob/1c98daf6e8bdf771ed3b17c947384fef4ed8d006/src/libfetchers/fetch-settings.hh#L23-L28

(it's a pair of `host=token`, which doesn't really care about the protocol -- at least, in the configuration)